docs/docs/snapshots.md

```diff
@@ -0,0 +1,52 @@
+# Chain State Snapshots
+
+As blockchains get longer, the process of syncing from the genesis block begins to take many hours, or even days to complete.
+In circumstances where a faster sync is required, various snapshots of the fetchd chain state data are available for download, to more quickly bootstrap a node.
+
+Snapshots are available for both mainnet and the most recent testnet.  The URLs can be obtained from the [network page](../networks/).  
+We aim to update snapshots on a daily basis.
```

Comment:
Sharing an issue I had with this - I was downloading a snapshot for osmosis, which took a few hours to complete, and a new one got produced mid download, killing my download and having to restart again.

Might be ok, but checking if we could have YYYYMMDD of the snapshot in the filename, and keep the last 2 or so (that'd break the link on the network page unless we can have a symlink pointing to latest or something), or for an easier option, just mention the approx time they're created here?

Reply:
If we add a timestamp into the filename, then we will need to implement housekeeping.  Whereas using different versions for the same file means that's taken care of by GCS.  

As of right now I'm producing snapshots every hour, to flush out any issues, but when I turn this down, I can put a daily creation time into the docs.

Resuming downloads is not going to be easy (possible) when piping through gunzip/tar as I have done.  But for people with unreliable internet connections, I'll see if I can come up with a resumable sequence which will tolerate version rotation.

Reply:
https://fetchai.atlassian.net/browse/ECO-1298 created

Reply:
adding summary from an external discussion on this:
- resumable downloads are low priority, not too concerned about user connection dying in the process.
- server killing current downloads by overwriting the file:  GCP should take care of this, it saves a few versions of previous files - So my initial problem should not apply here.